components/common/PrayerPoints.tsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import { Container, Card } from "react-bootstrap";
+import {FaPrayingHands} from 'react-icons/fa'
+import { IconContext } from "react-icons/lib";
+import { BsDownload } from "react-icons/bs";
+import Button from "react-bootstrap/Button";
+import { useTranslation } from "next-i18next";
+
+interface prayerProps {
+    prayerPoints: string[] // array of prayer points
+    title: string // Title that goes on top of card
+    featured?: boolean // is this component the "featured" component?
+    showImg?: boolean // show the image?
+    showSubtitle?: boolean // show the "Pray For" subtitle?
+}
+
+
+export default function PrayerPoints({prayerPoints, title, featured=false, showImg=false, showSubtitle=false}:prayerProps) {
+
+    const { t } = useTranslation("common");
+
+    const subtitle:string = t("prayerSummary.subtitle")
+    const view:string = t("prayerSummary.view")
+    const read:string = t("prayerSummary.read")
+
+    return (
+        <Container data-testid={"prayer-points-container"} className={"d-flex px-6" + (showSubtitle ? " justify-content-center" : " justify-content-start")} id="topic-prayer">
+            <Card className="my-4 shadow d-lg-flex flex-lg-row">
+                {/* temporary height for video, before we finalize videos */}
+                {showImg &&
+                    <Card.Img variant="top" src="/bamboo.jpeg" style={{height:'100%', maxWidth:'600px'}}></Card.Img>
```

Comment:
Would prefer we start looking at replacing style options with stuff like h-100. Not sure how that would apply with width, but it also has stuff like w-75, etc. We should test and try if we can do w-xl-50 etc

Reply:
Oops! yeah that was a bit of rust from not using bootstrap for a bit. I am trying my best to use the bootstrap classes as much as possible and just slipped up here. Please let me know if you catch anything else like this that can be changed to bootstrap.